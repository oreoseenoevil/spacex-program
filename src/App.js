import { Footer } from './Components/Layouts/Footer'
import { Header } from './Components/Layouts/Header'
import { Main } from './Components/Layouts/Main'
import { ProgramContextProvider } from './Context/ProgramContext'

function App() {

  return (
    <ProgramContextProvider>
      <div className="App">
        <Header name="SpaceX Launch Program" />
        <Main />
        <Footer name="Jessie Tarrosa" />
      </div>
    </ProgramContextProvider>
  )
}

export default App
