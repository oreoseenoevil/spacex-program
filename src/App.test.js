import { shallow } from 'enzyme'
import React from 'react'
import App from './App'

describe('<App />', () => {
  const wrapper = shallow(<App />)
  it('renders App component without crashing', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
