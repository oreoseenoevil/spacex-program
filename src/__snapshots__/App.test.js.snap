// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders App component without crashing 1`] = `"<div class=\\"App\\"><header class=\\"header\\"><h1 class=\\"headerName\\">SpaceX Launch Program</h1><h1 class=\\"headerName\\">Total Results: 0</h1></header><div class=\\"container\\"><div class=\\"sidebar\\"><div class=\\"card\\"><div style=\\"display:flex;width:100%;align-items:center;justify-content:space-between\\"><h3 style=\\"font-size:2rem;font-weight:700\\">Filters</h3><button class=\\"button active\\" style=\\"padding:.5rem 2rem;border-radius:.5rem;background:#fff;color:#000;border:1px solid #999;cursor:pointer\\">Reset</button></div><div class=\\"card\\"><h3 class=\\"filterName\\">Launch Year</h3><hr/><div class=\\"buttons\\"><button class=\\"button false\\">2006</button><button class=\\"button false\\">2007</button><button class=\\"button false\\">2008</button><button class=\\"button false\\">2009</button><button class=\\"button false\\">2010</button><button class=\\"button false\\">2011</button><button class=\\"button false\\">2012</button><button class=\\"button false\\">2013</button><button class=\\"button false\\">2014</button><button class=\\"button false\\">2015</button><button class=\\"button false\\">2016</button><button class=\\"button false\\">2017</button><button class=\\"button false\\">2018</button><button class=\\"button false\\">2019</button><button class=\\"button false\\">2020</button></div></div><div class=\\"card\\"><h3 class=\\"filterName\\">Successful Launch</h3><hr/><div class=\\"buttons\\"><button class=\\"button false\\">true</button><button class=\\"button false\\">false</button></div></div><div class=\\"card\\"><h3 class=\\"filterName\\">Successful Landing</h3><hr/><div class=\\"buttons\\"><button class=\\"button false\\">true</button><button class=\\"button false\\">false</button></div></div></div></div><div class=\\"tables\\"><div class=\\"centered\\"><h3>No Missions found.</h3></div></div></div><div class=\\"footer\\"><div class=\\"footerDetails\\"><span class=\\"spanOne\\">Developed By:</span><span class=\\"spanTwo\\">Jessie Tarrosa</span></div></div></div>"`;
